pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,62 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run easily on [Google Cloud Platform](https://cloud.google.com).
+
+<%= toc %>
+
+## Install the Agent on your own Google Compute Engine instance
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example, launch an instance using the [latest Ubuntu LTS image](https://console.cloud.google.com/launcher?cat=OS&q=ubuntu) and follow [our Ubuntu setup instructions](/docs/agent/ubuntu).
```

Comment:
I don't think they're separate clauses though? Do either make sense as standalone sentences? "To run the agent on your own Google Compute Engine instance." "Use whichever installer matches your instance type."